backend/psql/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const POSTGRES_UNAME = process.env.POSTGRES_UNAME || "algorand";
```

Comment:
rebase your branch with master branch. this should not be here. It's been moved to config.ts

Reply:
This file is still here???

Reply:
removed